airbyte-integrations/connectors/source-braintree/metadata.yaml

```diff
@@ -24,5 +24,6 @@ data:
   releaseStage: alpha
   supportLevel: community
   tags:
+    - language:low-code
```

Comment:
we need leave only one tag 

language:low-code or language:python should be presented in metadata file